addons/hr_expense/tests/test_expenses.py

```diff
@@ -410,3 +410,20 @@ def test_reset_move_to_draft(self):
 
         self.assertEqual(expense.state, 'done', 'Expense state must be done after payment')
         self.assertEqual(expense_sheet.state, 'done', 'Sheet state must be done after payment')
+
+    def test_expense_from_attachments(self):
+        self.env.user.employee_id = self.expense_employee.id
+        attachment = self.env['ir.attachment'].create({
+            'datas': b"R0lGODdhAQABAIAAAP///////ywAAAAAAQABAAACAkQBADs=",
+            'name': 'file.png',
+            'res_model': 'hr.expense',
```

Comment:
```suggestion
```
You don't need to specify it, it will be set in `create_expense_from_attachments()`